devito/ir/iet/nodes.py

```diff
@@ -888,8 +888,11 @@ def hoistable(self):
         return ()
 
     @property
-    def is_Useless(self):
-        return USELESS in self.properties
+    def useless(self):
+        for i in self.properties:
+            if i.name == 'useless':
```

Comment:
Rather than i.name ==, would be better to have a like `.useless` attribute?

Reply:
how do you mean? it is improvable, sure, but I don't get how you would do it? this is the ``useless`` attribute?

Reply:
ok